styles/RolePlayGateway/template/characters_profile_body.html

```diff
@@ -3,8 +3,11 @@
 <!-- INCLUDE roleplay_controls.html -->
 
 <!-- IF S_CAN_EDIT -->
-<div id="controls" style="float:right;">
+<div id="controls" class="quick-float-right">
```

Comment:
Oh, I see where this is located now on the sheet. I don't think I added this to the code, though I'm not positive how it got there.

Sorry. :X